cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/templates/etl/realtime/sources/JmsSource.java

```diff
@@ -258,4 +278,24 @@ public void setJmsProvider(JmsProvider provider) {
   public JmsProvider getJmsProvider() {
     return jmsProvider;
   }
+
+  /**
+   * Config class for {@link JmsSource}.
+   */
+  public static class JmsConfig extends PluginConfig {
+    @Name(JMS_DESTINATION_NAME)
+    @Description("Name of the destination to get messages")
+    private final String destinationName;
+
+    @Name(JMS_MESSAGES_TO_RECEIVE)
+    @Description("Max number messages should be retrieved per poll.")
+    @Nullable
+    private final Integer messagesToReceive;
```

Comment:
Yes for the messagesToReceive, I add the default in the JmsSource itself it is not being passed.

As for destination, need to be set.